frontend/src/components/SearchResult.js

```diff
@@ -5,22 +5,28 @@ import { IoIosAlarm } from 'react-icons/io';
 import { IoBarbellSharp } from 'react-icons/io5';
 import { useEffect, useState } from 'react';
 import { getRecipesWithMatchingIngredients } from './apiCalls';
+import ModalForSearch from './ModalForSearch';
 
 function SearchResult({ ingredients, searchBy }) {
-  const [recipe, setRecipe] = useState([]);
+  const [recipe, setRecipe] = useState(null);
+  const [show, setShow] = useState(false);
+  const handleShow = () => setShow(true);
```

Comment:
Ezt lehet érdemes egy useCallback-be rakni.

Reply:
próbálkozom , de nem tudom garantálni , hogy jövő héten kész lesz